kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -0,0 +1,96 @@
+server {
+    listen 80;
+
+    gzip                     on;
+    gzip_min_length          1000;
+    gzip_proxied             expired no-cache no-store private auth;
+    gzip_types               text/plain text/css text/javascript application/xml;
+    client_max_body_size     {{ max_upload_size }};
+
+
+    location /download/ {
+        alias /srv/storage/;
+    }
+
+    location = /upload/ {
+
+        access_by_lua_block {
+            -- Module nginx big upload support only content-type: "application/x-www-form-urlencoded"
+            check_content_type_header=ngx.req.get_headers()["Content-Type"]
```

Comment:
Please don't call variables like this. If you start the name with a verb, it sounds like a function. Should be `content_type_header`.